app/views/boards/show.html.erb

```diff
@@ -52,7 +52,7 @@ See doc/COPYRIGHT.rdoc for more details.
       <%= preview_link preview_board_topics_path(@board), 'message-form-preview', class: 'button' %>
       <%= link_to l(:button_cancel), "#", onclick: 'Element.hide("add-message")', class: 'button' %>
     <% end %>
-    <div id="preview" class="wiki"></div>
+    <div id="preview"></div>
```

Comment:
I am irritated that I find no `preview` *class* here... But it works, so I just say nothing...